src/main/java/org/sagebionetworks/web/client/SynapseJSNIUtils.java

```diff
@@ -14,8 +14,6 @@ public interface SynapseJSNIUtils {
 
   public void highlightCodeBlocks();
 
-  void loadSummaryDetailsShim();
```

Comment:
details-shim was removed in https://github.com/Sage-Bionetworks/SynapseWebClient/pull/5346

I missed these references to the module, so removing them here will make an error message in the console go away.